docs/context/ref/TransactionResult.md

```diff
@@ -0,0 +1,59 @@
+---
+id: TransactionResult
+sidebar_label: TransactionResult
+title: TransactionResult
+hide_title: true
+---
+# `TransactionResult`
+
+The result of any given create, update or delete intent.
```

Comment:
Heres a start on a deeper description. We'll also need to add something to the Intents overview doc describing the same. I'd suggest adding a new section at the bottom of https://fdc3.finos.org/docs/next/intents/spec for now, called something like:

> **Returning a status For transactional Intents**

```suggestion
`TransactionResult` is intended to be returned as an [`IntentResult`](../../api/Types#intentresult) by intents that create, retrieve, update or delete content or records in another application. Its purpose is to provide a status and message (where needed) for the transaction and MAY wrap a returned context object.
```